README.md

```diff
@@ -85,11 +85,12 @@ python broker_submission.py --eload 677 --project_accession PRJEB00001
 ### Data ingestion
 
 After validation and brokering are done, the data can be loaded into our databases and made publicly available.
-This involves three main steps:
+This involves four main steps:
 
 1. Loading submission metadata from ENA into EVAPRO (`metadata_load`)
-2. Accessioning VCF files (`accession`)
+2. Accessioning VCF files and making files public (`accession`)
 3. Loading into the variant warehouse (`variant_load`)
+4. Clustering in the current supported assembly, including remapping if necessary (`cluster`)
```

Comment:
couldn't decide whether to call this task `remap_and_cluster` or not... it seemed kind of wordy, but maybe just `cluster` is too confusing